### PR TITLE
Added code for Tax Slabs in java

### PR DESCRIPTION
**Pull request by raksha009**
**Fixes issue:** # 4533

Code added for slab problem of codechef
@AdiChat plz review my PR

### CODE REVIEW

**Comment by raksha009** [comment]
@arnavb plz review the PR.

**Comment by arnavb** [comment]
Why did you delete `break.c`?

File: code/online_challenges/src/codechef/SLAB/slab.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Slab {
```
Comment (arnavb) [inline code/online_challenges/src/codechef/SLAB/slab.java]: Please reformat this file with 4 space indentation. Also call the file `Slab.java`.

Comment (raksha009) [reply code/online_challenges/src/codechef/SLAB/slab.java]: But i formatted it using https://codebeautify.org/javaviewer#

**Comment by raksha009** [comment]
> Why did you delete `break.c`?

I deleted because that was made in a different PR and that Pr was made in master branch by mistakenly. It was my 1st commit.

File: code/online_challenges/src/codechef/SLAB/slab.java (hunk 1 above)
Comment (raksha009) [reply code/online_challenges/src/codechef/SLAB/slab.java]: What should I do!? Use a 4 space indendation or follow the formatting of online platform  https://codebeautify.org/javaviewer#. Plz suggest.

Comment (arnavb) [reply code/online_challenges/src/codechef/SLAB/slab.java]: @raksha009 Sorry. you are right, I didn't realize you can't change the settings there. Are you using an IDE? Try reformatting in that (For IntelliJ I know the option is there). If you want, I can reformat the file for you.

**Comment by arnavb** [comment]
@raksha009 But the problem is, if I merge this PR, it will delete the file since it shows that in the diff. Since your  other PR has been merged, revert 0b0a7da.

File: code/online_challenges/src/codechef/SLAB/slab.java (hunk 1 above)
Comment (raksha009) [reply code/online_challenges/src/codechef/SLAB/slab.java]: Yes, I am using eclipse. I did 4 sapce indentation initially, will redo it again. Thanks

**Comment by raksha009** [comment]
I tried searching online but i am not getting how to solve the conflicting issues. This conflict issues aren't shown on my terminal. Plz help.

**Comment by arnavb** [comment]
@raksha009 Instead of worrying about this PR, just close this and recreate a new one with your reformatted code. Do the same for your other PR with the merge conflict if you are unable to fix it:

```java
import java.io.*;
import java.util.*;

class Slab {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int t = s.nextInt();
        int a = 12500;
        int b = 25000;
        int c = 37500;
        int d = 50000;
        int e = 62500;

        while (t-- > 0) {
            long n = s.nextLong();
            long ans = 0;
            if (n <= 250000) {
                System.out.println(n);
            } else {
                if (n >= 250001 && n <= 500000) {
                    long div = (long) ((n - 250000) * 0.05);
                    System.out.println(n - div);
                } else if (n >= 500001 && n <= 750000) {
                    long div = (long) ((n - 500000) * 0.1);
                    System.out.println(n - a - div);
                } else if (n >= 750001 && n <= 1000000) {
                    long div = (long) ((n - 750000) * 0.15);
                    System.out.println(n - a - b - div);
                } else if (n >= 1000001 && n <= 1250000) {
                    long div = (long) ((n - 1000000) * 0.2);
                    System.out.println(n - a - b - c - div);
                } else if (n >= 1250001 && n <= 1500000) {
                    long div = (long) ((n - 1250000) * 0.25);
                    System.out.println(n - a - b - c - d - div);
                } else if (n > 1500000) {
                    long f = (long) ((n - 1500000) * 0.3);
                    System.out.println(n - a - b - c - d - e - f);
                }
            }
        }
    }
}

/*
TEST CASE

INPUT
2
600000
250000

OUTPUT
577500
250000
*/
```

**Comment by raksha009** [comment]
okk thanks

**Pull request closed without merging**